components/card/card.spec.ts

```diff
@@ -166,3 +168,33 @@ export class NzTestCardRtlComponent {
   @ViewChild(Dir) dir!: Dir;
   direction: Direction = 'rtl';
 }
+
+describe('card', () => {
```

Comment:
test suite `card` duplicated, what about renaming it?

Reply:
yeah you'r right, done 👍